pkgs/development/python-modules/torchrl/default.nix

```diff
@@ -0,0 +1,116 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, ninja
+, setuptools
+, wheel
+, which
+, cloudpickle
+, numpy
+, torch
+, ale-py
+, gymnasium
+, pygame
+, torchsnapshot
+, mujoco
+, moviepy
+, git
+, hydra-core
+, tensorboard
+, tqdm
+, wandb
+, packaging
+, tensordict
+, flaky
+, pytestCheckHook
+, pyyaml
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "torchrl";
+  version = "0.2.1";
+  format = "setuptools";
```

Comment:
```suggestion
  pyproject = true;
```

Reply:
Indeed it works !

Reply:
As mentioned in the comment, the `pyproject.toml` contains "unknown properties" which cause the nix build to crash if we set the format to `pyproject`.

Reply:
As mentioned in the comment, the `pyproject.toml` contains "unknown properties" which cause the nix build to crash if we set the format to `pyproject`.